dotnet-desktop-guide/net/winforms/whats-new/net60.md

```diff
@@ -0,0 +1,119 @@
+---
+title: What's new in Windows Forms .NET 6
+description: Learn about what's new in Windows Forms for .NET 6. Windows Forms. .NET provides new features and enhancements over .NET Framework.
+ms.date: 10/28/2021
+ms.topic: conceptual
+---
+
+# What's new for .NET 6 (Windows Forms .NET)
+
+Windows Forms for .NET 6 adds the following features and enhancements over .NET Framework.
+
+There are a few breaking changes you should be aware of when migrating from .NET Framework to .NET 6. For more information, see [Breaking changes in Windows Forms](/dotnet/core/compatibility/winforms).
+
+## New application bootstrap
+
+TODO
+
+## Updated templates for C#
+
+In line with [related changes in .NET workloads](../../sdk/6.0/csharp-template-code.md), Windows Forms templates for C# have been updated to support `global using` directives, file-scoped namespaces, and nullable reference types. Because a typical Windows Forms app consist of multiple types split across multiple files, for example, Form1.cs and Form1.Designer.cs, top-level statements are notably absent from the Windows Forms templates. However, the updated templates do include application bootstrap code. This can cause incompatibility if you target an earlier version of .NET.
+
+TODO: note about .NET Framework templates. @OliaG to provide text.
```

Comment:
As I mentioned - I think that we don't have to worry about that, but I'll confirm when installing on a clean VM.

Reply:
@RussKie I've confirmed - you can remove the ToDo about the .NET Framework templates. It's regressed in that they are optional components which are unchecked in the installer but they're still installing somehow on my clean VM.